### DISCUSSION TITLE
Bad Gateway

### DISCUSSION THREAD

**Discussion by mzapico** [ludeknovy/jtl-reporter#83] [Show and tell]
Hi Sir, I try to login into JTL reporter but unfortunately it gives me this error

![image](https://user-images.githubusercontent.com/87107607/125875588-4acc0d13-10c1-47ea-a3eb-4596b0df1c0c.png)

**Comment by ludeknovy** [comment]
Hi @mzapico, I guess the backend was not properly started. Could you please amend logs/errors from be service by running
`docker logs jtl-reporter-be` ?

**Reply by mzapico** [reply]
It's working now, but my locust did not Integrate on JtlReporter. I only have this view 

![image](https://user-images.githubusercontent.com/87107607/125926756-63e097c7-63f8-4f42-84dc-a4e54cf4c024.png)

**Reply by ludeknovy** [reply]
Did you follow the [docs](https://jtlreporter.site/docs/integrations/locust) ? Is there any error?

**Reply by mzapico** [reply]
Yes I did everything on docs but I didn't see any error

**Reply by ludeknovy** [reply]
If you dont see any error in locust script, it means the report was (uploaded)[https://github.com/ludeknovy/jtl-reporter/blob/master/scripts/jtl_listener.py#L134]. Please provide logs from backend, by running: `docker logs jtl-reporter-be`

**Reply by ludeknovy** [reply]
Ahh, Ok. You did not set the ‘JTL_API_TOKEN’ env var as stated here: https://jtlreporter.site/docs/integrations/locust

**Reply by ludeknovy** [reply]
Please put back `env=environment` in JtlListener constructor. You either need to set the JTL_API_TOKEN as environ variable, in unix environments you do it by `export JTL_API_TOKEN=<value>`, not sure how to do in in Windows, sorry. Or you can set it here https://github.com/ludeknovy/jtl-reporter/blob/master/scripts/jtl_listener.py#L43 if you dont wont to set it as env var.

If you dont have your API token yet, here is a guide how to generate it: https://jtlreporter.site/docs/guides/administration/api-token

this comment relates to previously deleted post by @mzapico.

**Reply by ludeknovy** [reply]
@mzapico did you solve it?

**Reply by mzapico** [reply]
Yes, Sir JltReporter is working. Thank you for assitance

**Reply by ludeknovy** [reply]
Splendid. Glad you made it work. Could you please mark this question as "answered"?

**Reply by mzapico** [reply]
Yes Sir Thank you again
